docs/Installing.asciidoc

```diff
@@ -1198,6 +1198,55 @@ sudo zypper rm postgresql$oldver-server postgresql$oldver-contrib postgresql$old
 sudo -u postgres rm -r /var/lib/pgsql/data.$oldver
 ----
 
+== Working on database-related performance problems
+Without extra setup, PostgreSQL already gathers many statistics, checkout
+https://www.postgresql.org/docs/current/monitoring-stats.html[the official documentation].
+
+=== Enable further statistics
+These statistics help to identify the most time-consuming queries.
+
+1. Configure the PostgreSQL extension `pg_stat_statements`, see example on
+   https://www.postgresql.org/docs/current/pgstatstatements.html[the official documentation].
+2. Ensure the extension library is installed which might be provided by a
+   separate package (e.g. `postgresql14-contrib` for PostgreSQL 14 on openSUSE).
+3. Restart PostgreSQL.
+4. Enable the extension via `CREATE EXTENSION pg_stat_statements`.
+
+==== Useful queries
+Simply query the table `pg_stat_statements`. Use `\x` in `psql` for extended
+mode or `substring()` on the `query` parameter for readable output. The columns
+are explained in the previously mentioned documentation. Here an example to show
+similar queries which are most time-consuming:
+
+```
+SELECT
+  substring(query from 0 for 250) as query_start, sum(calls) as calls, max(max_exec_time) as max_exec_time,
+  sum(total_exec_time) as total_exec_time, sum(rows) as rows
+  FROM pg_stat_statements group by query_start ORDER BY total_exec_time DESC LIMIT 10;
+```
```

Comment:
Just add a line something like "consider resetting statistics after significant database changes and query plans"

Reply:
After significant schema changes consider resetting query statistics (`SELECT pg_stat_statement_reset()`) and checking the query plans (`EXPLAIN (ANALYZE, BUFFERS) ...`) for the slowest queries showing up afterwards.